t/Currency.t

```diff
@@ -111,13 +111,19 @@ ddg_spice_test(
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
-    # using value keyword
+    # using value, price keyword
     'bitcoin value' => test_spice(
         '/js/spice/currency/1/xbt/xbt',
         call_type => 'include',
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
+    'bitcoin price' => test_spice(
+        '/js/spice/currency/1/xbt/xbt',
+        call_type => 'include',
+        caller => 'DDG::Spice::Currency',
+        is_cached => 0
+    ),
```

Comment:
Its working for me in the duckpan server

![screenshot from 2017-04-25 14-47-18](https://cloud.githubusercontent.com/assets/25108385/25377838/41a38c1a-29c6-11e7-93e0-851bdd9e1f81.png)